Trimble.Modus.Components/Converters/FloatingButtonValueConverter.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Trimble.Modus.Components.Enums;
+
+namespace Trimble.Modus.Components.Converters
+{
+    public class FloatingButtonValueConverter : IValueConverter
```

Comment:
Not used anywhere. Do you still need this?

Reply:
No, Removed